### PR TITLE
[IMP] Add an option on res.partner records to define partner as manuf…

### PR DESCRIPTION
**Pull request by qtheuret**
…acturer and filter on these partners when search manufacturers list

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
Hi @qtheuret 

Could you have a look to the pylint? you still have a small issue.

**Comment by qtheuret** [comment]
@lreficent I also fixed Pylint issues with other modules.

File: product_brand/__manifest__.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
     'summary': "Product Brand Manager",
     'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA'
               ', MONK Software, Odoo Community Association (OCA)'
-              ', SerpentCS Pvt. Ltd.',
+              ', SerpentCS Pvt. Ltd., Amaris',
```
Comment (LoisRForgeFlow) [inline product_brand/__manifest__.py:16]: Fixing the pylint doesn't make you author of the module, please remove this and add yourself to the contributors sections if you wish, however you haven't change/improved any functionality of the module, IMHO you shouldn't either. 

Same applies for all the copyright you added on the rest of files.

File: product_brand/__init__.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
-# -*- coding: utf-8 -*-
 # © 2009 NetAndCo (<http://www.netandco.net>).
 # © 2011 Akretion Benoît Guillot <benoit.guillot@akretion.com>
 # © 2014 prisnet.ch Seraphine Lantible <s.lantible@gmail.com>
 # © 2016  Serpent Consulting Services Pvt. Ltd.
 # © 2018 Daniel Campos <danielcampos@avanzosc.es>
+# © 2018 Amaris - Quentin Theuret <quentin.theuret@amaris.com)
```
Comment (LoisRForgeFlow) [inline product_brand/__init__.py:6]: I would remove all the copyrights from the inits files.

File: product_manufacturer/views/res_partner_view.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <field name="res_model">res.partner</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">kanban,tree,form</field>
+        <field name="domain">[('manufacturer', '=', True)]</field>
+    </record>
+
+    <menuitem
+        id="menu_res_partner_manufacturer"
+        action="action_res_partner_manufacturer"
+        parent="" />
```
Comment (LoisRForgeFlow) [inline product_manufacturer/views/res_partner_view.xml:26]: I don't think a top level menu for this is justified. Top level menus are for whole applications: mrp, accounting, stock... You can add this menu below "Vendors" on the purchase app.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Thanks for fixing! 

See my comments in line.

**Pull request closed without merging**